src/main/scala/com/gd/twitteranalytics/reports/ActiveUserReportProcessor.scala

```diff
@@ -0,0 +1,65 @@
+package com.gd.twitteranalytics.reports
+
+import org.apache.spark.sql.expressions.Window
+import org.apache.spark.sql.expressions.scalalang.typed.{count => typedCount}
+import org.apache.spark.sql.functions.{count,lit, row_number}
+import org.apache.spark.sql.{DataFrame, Dataset, Encoders, SparkSession}
+
+object ActiveUserReportProcessor {
+
+  def getReportWithSqlProcessing(spark: SparkSession, tweetStatusDf: DataFrame,reportDate:String): DataFrame = {
+    tweetStatusDf.createOrReplaceTempView("status")
+    val activeUsersIdAndLocation = """SELECT
+                                        location,
+                                        id,
+                                        COUNT(id) AS frequency
+                                      FROM status
+                                      GROUP BY location,
+                                               id
+                                      HAVING frequency >= 10"""
+    spark.sql(activeUsersIdAndLocation).withColumn("date", lit(reportDate)).createOrReplaceTempView("activeUsers")
```

Comment:
please apply the following formatting for readability of such long sequences:
```scala

spark
    .sql(activeUsersIdAndLocation)
    .withColumn("date", lit(reportDate))
    .createOrReplaceTempView("activeUsers")

```

Reply:
Fixed..